apps/agreement/contracts/test/mocks/disputable/AragonAppFeesCashierMock.sol

```diff
@@ -41,7 +42,8 @@ contract AragonAppFeesCashierMock is IAragonAppFeesCashier, EtherTokenConstant,
         }
     }
 
-    function payAppFees(bytes32, bytes) external {
+    function payAppFees(bytes32, bytes) external payable {
+        require(msg.value == 0, ERROR_NON_ZERO_VALUE);
```

Comment:
Maybe we should have a different mock method for this that requires ETH to be sent?

Otherwise this precludes token fees at the moment.

Reply:
True, let me know @bingen if there is a reason for this change, otherwise we can address it in #1212

Reply:
Just in case https://github.com/aragon/aragonOS/pull/593